master.yaml

```diff
@@ -2,49 +2,80 @@
 # mayaonline can parse this yaml to obtain tests specific to applications (based on tag) & populate test params,
 # fulfill pre-requisites etc, before deployment. The params are explicitly typed to prompt right values from user.
 # Test parameters can be of different kinds such as ENV, configmap etc., with their own properties.
-# '@userdata' is a placeholder which is either autofilled/userfilled. If empty, use defaults. The test has tags which
-# will help with categorization & application-mapping
+# The test has tags which will help with categorization & application-mapping.
 
 
 tests:
-  - name: pod_kill
-    test_tags: ['mysql', 'pumba']
-    specPath: ./apps/percona/tests/mysql_data_persistence/run_litmus_test.yaml
+  - name: openebs-volume-replica-failure
+    test_tags: ['mysql', 'openebs']
+    specPath: ./apps/percona/chaos/openebs_volume_replica_failure/run_litmus_test.yaml
     patch:
-    - name: 'Server IP'
+    - name: 'APPLICATION NAMESPACE'
```

Comment:
I know, there's a reason behind using all uppercase here, but can it happen for any case where mixed case will be better?

Like: Can we consider 'Server IP' over 'SERVER IP' or 'server ip' for any case?

Reply:
Can be done.

Reply:
@a4abhishek , have changed the name descriptors of the tests. Note that the actual ENV names are still maintained under values section.